### PR TITLE
Fix #280: removed todo, require ext arg

### LINKED ISSUE #280: pkcli.github glob for checking for previous clone is unnecessary
The code is 
```python
    def _prev_backup(self, base, ext=_TXZ):
        # POSIT: timestamp Backup
        b = pkio.sorted_glob(f"../*/{base}{ext}")
        return b[-1] if b else []
```

In the case of zgoubi git we have 
base = radiasoft-zgoubi.git
ext = "*"

That matches both radiasoft-zgoubi.git and radiasoft-zgoubi.github.io.git.

So we cp in radiasoft-zgoubi.github.io.git
https://github.com/radiasoft/pykern/blob/0f782f55bb0eee315f9b675f10a3e9bc0567e074/pykern/pkcli/github.py#L447

And the chdir doesn't work because radiasoft-zgoubi.git doesn't exist
https://github.com/radiasoft/pykern/blob/0f782f55bb0eee315f9b675f10a3e9bc0567e074/pykern/pkcli/github.py#L448

### PR DESCRIPTION
**Pull request by e-carlin**
In the event the glob matches multiple values. For example, radiasoft-zgoubi.git and radiasoft-zgoubi.github.io.git. Then use the first result which will be the shorter and more "specific" result.

- [ ] Require test?
   - N
- [ ] Security implications?
    - N
- [ ] UI changes reviewed?
    - N
- [ ] Backwards compatible?
    - I hope... hard to test
- [ ] Require doc changes?
    - N
- [ ] Add [DesignHints](https://github.com/radiasoft/pykern/wiki/DesignHints)?
    - N

@robnagler this seems rather challenging to test and I'm concerned about edge cases I'm not thinking of. Any thoughts?

### CODE REVIEW

**Comment by e-carlin** [comment]
On closer thought. Maybe the "*" glob can be removed entirely. What case is that covering?

**Comment by e-carlin** [comment]
I updated to remove the glob. I don't think there was any use for it.

File: pykern/pkcli/github.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def _repo(self, repo):
 
         def _clone(suffix):
             base = bd + suffix
-            prev = self._prev_backup(base, ext="*")
+            prev = self._prev_backup(base, ext="")
```
Comment (robnagler) [inline pykern/pkcli/github.py:430]: isn't it needed for wiki?

Comment (e-carlin) [reply pykern/pkcli/github.py:430]: I don't believe so. In the wiki case we'll look for 'radiasoft-<repo>.wiki.git' which will work. Rigth?
